packages/use-i18n/src/index.js

```diff
@@ -235,15 +250,15 @@ const I18nContextProvider = ({
       currentLocale,
       dateFnsLocale,
       datetime,
-      formatNumber,
       formatList,
+      formatNumber,
       loadTranslations,
-      locales,
       namespaceTranslation,
+      namespaces,
       relativeTime,
       relativeTimeStrict,
-      setLocales,
       setTranslations,
+      supportedLocales,
```

Comment:
What the difference between supportedLocales and locales ?

Reply:
Before we use a state and it's not used anymore